back-end/confirmHealth.js

```diff
@@ -1,3 +1,25 @@
-const confirmHealth = (snack) => {};
+const confirmHealth = (snack) => {
+    const { name, fiber, protein, added_sugar, is_healthy, image } = snack;
+
+    if (typeof fiber !== 'number' ||
+        typeof protein !== 'number' ||
+        typeof added_sugar !== 'number'
+    ) {
+        return null
+    } else {
+        if (fiber > 5 && added_sugar < 5 || 
+            protein > 5 && added_sugar < 5 ||
+            protein > 5 && fiber > 5 && added_sugar < 5
+            ) {
+            return true;
+        } else if (fiber > 5 && added_sugar > 5 ||
+            protein > 5 && added_sugar > 5 ||
+            protein > 5 && fiber > 5 && added_sugar > 5
+            ) {
+            return false;
+        }
```

Comment:
these checks are a great place a for a little helper function to make our code more expressive